spec/fixtures/events.yml

```diff
@@ -7,3 +7,13 @@ jane_website_agent_event:
   user: jane
   agent: jane_website_agent
   payload: <%= { :title => "foo", :url => "http://foo.com" }.to_json.inspect %>
+
+url_event:
+  user: bob
+  agent: bob_website_agent
```

Comment:
I don't think these events would actually be created by a website agent, right?  Perhaps you can just `create` them in simple_meta_agent_spec.rb?